examples/mobile/UIComponents/js/app.js

```diff
@@ -36,12 +38,21 @@
 		dropDownMenuWidget.open();
 	}
 
-	document.addEventListener("pagebeforeshow", function () {
+	document.addEventListener("pageshow", function () {
```

Comment:
This handler can be as named function `onPageShow` insted anonymous function.

Reply:
done, also aded onPageHide handler